### PR TITLE
Initial file for githubteacher bio

### PR DESCRIPTION
**Pull request by githubteacher**
Adding the file to address #10

### CODE REVIEW

File: bios/githubteacher-bio.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## A little about me
+
+- I live in NC
```
Comment (githubteacher) [inline bios/githubteacher-bio.md]: Where do you live in NC?

**Comment by githubteacher** [comment]
:+1: That is a great idea! I will add it now!

File: bios/githubteacher-bio.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## A little about me
+
+- I live in Fuquay, NC
+- I love Arduino and gardening
+- I have two dogs
```
Comment (ksairamesh) [inline bios/githubteacher-bio.md:5]: What breed are your dogs?

**Comment by socialac** [comment]
New York is a cool place to live

**Pull request merged**